### PR TITLE
Fix process kill  and added env for proxy. Fixes LambdaTest/tunnel-operations#101

### PR DESCRIPTION
**Pull request by kanhaiya15**

### CODE REVIEW

File: lib/tunnel.js [hunk 1]
```diff
@@ -466,24 +466,44 @@ function killRunningProcess_(that, fnCallback) {
...
+      var data = '';
+      var reqOptions = {
+        hostname: parseURL.hostname,
+        port: parseURL.port,
+        path: parseURL.path,
+        method: 'DELETE',
+        headers: {
+          'Content-Type': 'application/json',
+          'x-ml-auth': 'LambdaTest',
+          'Content-Length': data.length
```
Comment (psych0der) [inline lib/tunnel.js:481]: You shouldn't add content length yourself. Let the library handle that. You can use got or axios for that

File: lib/tunnel.js [hunk 2]
```diff
@@ -857,5 +877,30 @@ function clearTimeout_(timeoutId) {
...
+      setTimeout(function() {
+        try {
+          process.kill(that.proc.pid);
+        } catch (err) {}
+        return fnCallback();
+      }, 500);
+    }
+    return fnCallback();
+  });
+  if (os.platform() == 'win32') {
```
Comment (psych0der) [inline lib/tunnel.js:898]: Why is this condition in addition to an API call?
These should be used only when exit call sent non 200 response.

Comment (kanhaiya15) [reply lib/tunnel.js:898]: Yes sir i did the same please see https://github.com/LambdaTest/node-tunnel/pull/5/files/447ed29bfbf7712c986a19b923db21cb83467de6#diff-7cb25e6733372586a7d0530feba6f8aeR487

File: lib/tunnel.js (hunk 1 above)
Comment (kanhaiya15) [reply lib/tunnel.js]: Updated sir removed header

**Pull request merged**